chall05/jtrancos.c

```diff
@@ -0,0 +1,29 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   jtrancos.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: jtrancos <marvin@42.fr>                    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2020/04/16 13:01:08 by jtrancos          #+#    #+#             */
+/*   Updated: 2020/04/16 13:36:56 by jtrancos         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stdio.h>
+
+int	ft_ie_except_after_c(char* str)
+{
+	int i;
+
+	i = 0;
+	while (str[i])
+	{
+		if (str[i] == 'e' && str[i + 1] == 'i' && str[i - 1] != 'c')
+			return (0);
+		if (str[i] == 'i' && str[i + 1] == 'e' && str[i - 1] == 'c')
+			return (0);
+		i++;
+	}
```

Comment:
You could have used `strstr` to iterate on the character combination ;)

Reply:
Thank you for the suggestion and the evaluation :)